packages/itwinui-react/src/core/utils/functions/dom.ts

```diff
@@ -52,3 +52,43 @@ export const getTranslateValues = (element: HTMLElement | null | undefined) => {
 
   return [matrix.m41, matrix.m42];
 };
+
+/**
+ * Accepts a handler which gets called when focus leaves the element
+ * as a result of a keypress (rather than mouse clicks).
+ *
+ * Must be passed to `keydown` to ensure it gets called before `blur` event.
+ *
+ * @example
+ * onKeyDown={handleFocusOut(() => {})}
+ */
+export const handleFocusOut =
```

Comment:
Why do we need this handler?

Reply:
i added it for closing the popover when tabbing out of it, but it looks like my implementation has changed significantly and this is not working anymore.

will investigate

Reply:
so it looks like `handleFocusOut` was not working correctly and is not needed if [`FloatingFocusManager`](https://floating-ui.com/docs/floatingfocusmanager) is used, because the latter will properly return focus to the trigger and close the popover.

https://github.com/iTwin/iTwinUI/assets/9084735/8f5782f6-76b5-4947-bd70-9cd456789eab

removed in [`29e37fd`](https://github.com/iTwin/iTwinUI/pull/1506/commits/29e37fd89418072739117ffae30bb20b83106c45).

i also think i ended up in some weird in-between state in storybook where `FloatingFocusManager` was being used even after i removed it. all the more reason we need to get rid of storybook asap